autoendpoint/src/settings.rs

```diff
@@ -17,6 +17,7 @@ pub struct Settings {
     pub scheme: String,
     pub host: String,
     pub port: u16,
+    pub endpoint_url: String,
```

Comment:
minor nit: could also do `Option<String>` defaulting to `None`

Reply:
Yeah, I thought about that. I wasn't sure how well `Settings` would deal with that since I remember it being kind of a hassle, so I went the cheap route for now.

Reply:
It works as you'd expect, see `statsd_host` in autoendpoint-rs  we use it elsewhere. No big deal though